src/pybind/mgr/dashboard_v2/controllers/cluster_configuration.py

```diff
@@ -10,22 +10,8 @@
 @ApiController('cluster_conf')
 @AuthRequired()
 class ClusterConfiguration(RESTController):
-    def list(self, service=None, level=None):
```

Comment:
Is this being removed because it's not being used by the dashboard frontend?

Reply:
yes.